xcube/core/store/descriptor.py

```diff
@@ -181,7 +228,13 @@ def _get_base_type_specifier(cls) -> TypeSpecifier:
 class VariableDescriptor:
     """
     A descriptor for dataset variable represented by xarray.DataArray instances.
-    They are part of dataset descriptor for an gridded, N-dimensional dataset represented by xarray.Dataset.
+    They are part of dataset descriptor for an gridded, N-dimensional dataset represented by
+    xarray.Dataset.
+
+    :param name: The variable name
+    :param dtype: The data type of the variable.
```

Comment:
I *think* that we're using `numpy.sctypeDict` format for type definitions, but would be glad of confirmation from you and @forman . Whatever format we're using, it should be specified here.

```suggestion
    :param dtype: The data type of the variable. Must be a string from `numpy.sctypeDict.keys()`
```

Reply:
As our software just reads and passes on this bit of information, I think it is okay if we don't make any assumptions about this. It may probably even be better, as this allows for different data stores to interpret this differently.

Reply:
Fair enough, makes sense -- but I think that that should also be made explicit with a note along the lines of "Syntax of data type specifiers is defined by the data store.".